sympy/solvers/solveset.py

```diff
@@ -3213,9 +3213,9 @@ def _solve_using_known_values(result, solver):
     new_result_complex, solve_call2, cnd_call2 = _solve_using_known_values(
         old_result, solveset_complex)
 
-    # when `total_solveset_call` is equals to `total_conditionset`
-    # means solvest fails to solve all the eq.
-    # return conditionset in this case
+    # when `total_solveset_call` equals to `total_conditionset`
+    # implies `solveset` failed to solve all the equations
+    # returning conditionset in this case
```

Comment:
Neither of these version is coherent English. I suggest:

If total_solveset_call is equal to total_conditionset then solveset failed to solve all of the equations. In this case we return a ConditionSet here.

The backticks are not needed in comments.